create-dev-env.sh

```diff
@@ -269,7 +267,7 @@ else
     -p 5432:5432 \
     --network gitops-net \
     -d \
-    postgres:13 \
+    $POSTGRESQL_DATABASE:13 \
```

Comment:
`postgres:13 \` is the container image name/tag, not the database name, so it's fine to leave it as `postgres:13`